rails_multisite.gemspec

```diff
@@ -17,9 +17,9 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = RailsMultisite::VERSION
 
-  gem.required_ruby_version = ">=2.4.0"
+  gem.required_ruby_version = ">=2.5.0"
```

Comment:
To reflect minimum version for Ruby is now 2.5 (was an oversight of my previous PR actually :sweat_smile:)